src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -255,7 +255,7 @@ private String getIssue(String filename) throws IOException, InterruptedExceptio
             while ((issue = br.readLine()) != null) {
                 issueId = issue;
             }
-            return issueId;
+            return issueId.trim().length() > 0 ? issueId : null;
```

Comment:
safer to use StringUtils.isEmpty (to manage issueId == null)

Reply:
Fixed

Reply:
[StringUtils.html#trimToNull](https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#trimToNull(java.lang.String)) seems to be a better option.  What do you think?